cluster/ci/jobs/codeready-toolchain-mirror.yaml

```diff
@@ -0,0 +1,50 @@
+kind: List
```

Comment:
The CronJob does not need to be here when the ProwJobs are set up.

Reply:
done in 6e0a47f